### PR TITLE
Get tests passing

### PR DESCRIPTION
**Pull request by odow**
<s>Tests still not passing on Windows. </s>

I'll dig into the file issue.

Edit: removed old test ouput

### CODE REVIEW

File: src/jamsd_solve.jl [hunk 1]
```diff
@@ -74,14 +74,14 @@ function jamsd_setup_gams()
 
     gams_dir = mktempdir(pwd())
     cur_dir = gams_dir
-    if isdir(gams_dir)
-        try
-            rm(gams_dir, recursive=true, force=true)
-            mkdir(gams_dir)
```
Comment (odow) [inline src/jamsd_solve.jl:80]: @xhub what is the above meant to do?

To me it looks like it makes `gams_dir`, then attempts to delete the directory, then remake it?

Comment (xhub) [reply src/jamsd_solve.jl:80]: That should be deleted. I just changed the behavior to create a temporary directory every time. Before ``gams_dir'' was a constant string. Hence, the need to delete it if it already existed. This code is now obsolete and should be deleted

**Comment by odow** [comment]
I remove the BARON test because I think I needed a license?

**Review by xhub — APPROVED** [review]
Thanks a lot for all the changes! Looks very nice!

**Comment by odow** [comment]
Also:

The `/test` directory fills up with `equ0.dot` files. But I can't find where they are being made. The same with `dummy.lst`. Should I just clean up after the tests have run?

**Comment by xhub** [comment]
It's fine to disable BARON, I was able to use it on all my setup, but I guess it depends on the type of license.

The equ*.dot files are useful only when I wanna debug, I've uploaded a new version of the dll that should not produce those.
If we add curdir=$substr to the gams function call, the dummy.lst files should be create inside the temporary directory and hopefully deleted.

I'm good to merge this PR if you are satisfied with it.

**Comment by odow** [comment]
I'm good to merge.

**Pull request merged**